torch/_inductor/fx_passes/binary_folding.py

```diff
@@ -34,7 +34,7 @@ def mark_mixed_dtype_conv(conv):
 
     if not (
         conv_user.target == prims.convert_element_type.default
-        and conv_user.args[1] == conv_dtype
+        and conv_user.args[1] not in (torch.float16, torch.bfloat16, torch.float32)
```

Comment:
@eellison , now for the conv+binary folding, we will always do the conv+binary folding as fp32, and then convert fp32 conv to original dtype conv, then call the original  ```prims.convert_element_type.default``` to make sure we don't change the folding patterns' output type, is it ok?